### PR TITLE
Generalize commit drag element

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Dragging a commit is not unique to cherry-picking thus the drag element should just be "Commit"

### Screenshots

Notes: no-notes

### CODE REVIEW

File: app/src/models/drag-drop.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { GitHubRepository } from './github-repository'
+
+/**
+ * This is a type is used in conjunction with the drag and drop manager to to
+ * store and specify the types of data that are being dragged
+ *
+ * Thus, using a `|` here would allow us to specify multiple types of data that
+ * can be dragged.
+ */
+export type DragData = ReadonlyArray<Commit>
```
Comment (tidy-dev) [inline app/src/models/drag-drop.ts:11]: Didn't mean to catch `DragData` in this PR.. but needed for next one anyways

File: app/src/ui/app.tsx [hunk 2]
```diff
@@ -2338,9 +2338,9 @@ export class App extends React.Component<IAppProps, IAppState> {
     const { currentDragElement } = this.state
     const isCherryPickCommitBeingDragged =
```
Comment (tidy-dev) [inline app/src/ui/app.tsx:2339]: Just saw that I missed renaming `isCherryPickCommitBeingDragged` to not be cherry-pick specific. But, changing this line in next PR anyway so I will catch it there

**Review by sergiou87 — APPROVED** [review]
Looks good! I like where this is going 😄 ✨

**Review by garywheeler69 — COMMENTED** [review]
Hu

**Pull request merged**